R/utils_gcp.R

```diff
@@ -0,0 +1,102 @@
+# Semi-automated tests of Google Cloud Storage integration live in tests/gcp/. # nolint
+# These tests should not be fully automated because they
+# automatically create S3 buckets and upload data,
+# which could put an unexpected and unfair burden on
+# external contributors from the open source community.
+# nocov start
+gcp_gcs_exists <- function(
+  key, 
+  bucket = googleCloudStorageR::gcs_get_global_bucket(), 
```

Comment:
I think it's cool that `googleCloudStorageR` lets you have a default bucket.

Reply:
I use it myself a lot and got tired of typing out bucket names all the time ;)